app/assets/javascripts/discourse-common/addon/lib/legacy-resolver.js

```diff
@@ -0,0 +1,336 @@
+/* eslint-disable no-undef */
```

Comment:
```suggestion
/* global Ember */
```

casts less of a wide net 🙂